### PR TITLE
Manage users in InfluxDB Clustered

### PR DESCRIPTION
**Pull request by sanderson**
Related to https://github.com/influxdata/DAR/issues/436

- [x] Rebased/mergeable

### CODE REVIEW

**Review by jenniferplusplus — APPROVED** [review]

File: content/influxdb/clustered/admin/users/_index.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+`AppInstance` resource. Administrative access lets user perform actions like
+creating databases and tokens.
+
+{{% note %}}
+#### Users versus database tokens
+
+All _users_ have administrative access to your cluster and can perform
+administrative actions in your InfluxDB cluster.
+_Database tokens_ authorize read and write access to databases in your InfluxDB
+cluster. One does not need to be a users to read and write data in your cluster,
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/_index.md:28]:
```suggestion
cluster. A person or client doesn't need to be a user to read and write data in your cluster,
```

File: content/influxdb/clustered/admin/users/add.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+    needs administrative access to your InfluxDB cluster.
+
+    **Refer to your identity provider's documentation for information about
+    adding users:**
+
+    - [Keycloak: Creating users {{% icon "export" %}}](https://www.keycloak.org/docs/latest/server_admin/#proc-creating-user_server_administration_guide)
+    - [Microsoft Entra ID: How to create, invite, and delete users {{% icon "export" %}}](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/add-users)
+    - [Auth0: Team member management {{% icon "export" %}}](https://auth0.com/docs/get-started/auth0-teams/team-member-management)
+
+2.  Add the newly added user to your InfluxDB `AppInstance` resource.
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:28]:
```suggestion
2.  Add the user to your InfluxDB `AppInstance` resource.
```

File: content/influxdb/clustered/admin/users/add.md [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+[Keycloak](#)
+[Auth0](#)
+[Microsoft Entra ID](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+{{% code-callout "keycloak" "green" %}}
+{{% code-placeholders "KEYCLOAK_(HOST|REALM|USER_ID)" %}}
+
+<!--pytest.mark.skip-->
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:69]:
```suggestion
```
It's only necessary to skip executables (`bash`, `sh`, `python`)

File: content/influxdb/clustered/admin/users/add.md [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+
+```yaml
+admin:
+  # The identity provider to be used e.g. "keycloak", "auth0", "azure", etc
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:214]:
```suggestion
  # The identity provider to be used (such as "keycloak", "auth0", or "azure")
```

File: content/influxdb/clustered/admin/users/add.md [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+  # Note for Azure Active Directory it must be exactly "azure"
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:215]:
```suggestion
  # Note, use "azure" for Azure Active Directory
```

File: content/influxdb/clustered/admin/users/add.md [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+---
+
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+
+{{% code-callout "auth0" "green" %}}
+{{% code-placeholders "AUTH0_(HOST|USER_ID)" %}}
+
+<!--pytest.mark.skip-->
+
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:251]:
```suggestion
```

Not needed.

File: content/influxdb/clustered/admin/users/add.md [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+---
+
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+
+{{% code-callout "azure" "green" %}}
+{{% code-placeholders "AZURE_(USER|TENANT)_ID" %}}
+
+<!--pytest.mark.skip-->
+
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:284]:
```suggestion
```

File: content/influxdb/clustered/admin/users/add.md [hunk 8]
```diff
@@ -0,0 +1,358 @@
...
+{{% code-callout "auth0" "green" %}}
+{{% code-placeholders "AUTH0_(HOST|USER_ID)" %}}
+
+<!--pytest.mark.skip-->
+
+```yaml
+admin:
+  # The identity provider to be used e.g. "keycloak", "auth0", "azure", etc
+  # Note for Azure Active Directory it must be exactly "azure"
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:255]:
```suggestion
  # Note, use "azure" for Azure Active Directory.
```

File: content/influxdb/clustered/admin/users/add.md [hunk 9]
```diff
@@ -0,0 +1,358 @@
...
+
+{{% code-callout "azure" "green" %}}
+{{% code-placeholders "AZURE_(USER|TENANT)_ID" %}}
+
+<!--pytest.mark.skip-->
+
+```yaml
+admin:
+  # The identity provider to be used e.g. "keycloak", "auth0", "azure", etc
+  # Note for Azure Active Directory it must be exactly "azure"
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/add.md:288]:
```suggestion
  # Note, use "azure" for Azure Active Directory.
```

File: content/influxdb/clustered/admin/users/remove.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+menu:
+  influxdb_clustered:
+    name: Remove a user
+    parent: Manage users
+weight: 201
+---
+
+Remove a user with administrative access from your InfluxDB cluster:
+
+1.  Remove or disable the user in your identity provider.
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/remove.md:15]:
```suggestion
1.  Remove or deactivate the user in your identity provider.
```

File: content/influxdb/clustered/admin/users/remove.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+{{% tab-content %}}
+<!----------------------------- BEGIN AppInstance ----------------------------->
+
+If editing your `AppInstance` resource directly, remove the user from the list
+of users in the `spec.package.spec.admin.users` field in your `myinfluxdb.yml`
+configuration file--for example:
+
+<!--pytest.mark.skip-->
+
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/remove.md:45]:
```suggestion
```

File: content/influxdb/clustered/admin/users/remove.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+<!------------------------------ END AppInstance ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!--------------------------------- BEGIN Helm -------------------------------->
+
+If using the InfluxDB Clustered Helm chart, remove the user from the list of
+users in the `admin.users` field in your in your `values.yaml`--for example:
+
+<!--pytest.mark.skip-->
+
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/remove.md:75]:
```suggestion
```

File: content/influxdb/clustered/admin/users/remove.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+  --namespace influxdb
+```
+
+{{% /code-tab-content %}}
+  {{< /code-tabs-wrapper >}}
+
+{{% note %}}
+After you complete step 1 above, the removed user no longer has administrative
+access to your InfluxDB cluster. However, they should still be removed from
+your `AppInstance` resource.
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/users/remove.md:134]:
```suggestion
access to your InfluxDB cluster.
However, you should still remove them from your `AppInstance` resource.
```

File: content/influxdb/clustered/install/configure-cluster/use-helm.md [hunk 14]
```diff
@@ -816,11 +818,11 @@ Replace the following:
 {{% /code-tab-content %}}
 {{< /code-tabs-wrapper >}}
 
-##### Adding users
+##### Add users
 
 Finally, add all the users you wish to have access to use `influxctl`.
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster/use-helm.md:823]:
```suggestion
Finally, add the users you wish to have access to use `influxctl`.
```

File: content/influxdb/clustered/install/configure-cluster/use-helm.md [hunk 15]
```diff
@@ -816,11 +818,11 @@ Replace the following:
...
 Update the `admin.users` field with a list of these users.
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster/use-helm.md:824]:
```suggestion
Update the `admin.users` field with a list of the users.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Looks good. Just a few suggestions.

**Review by jstirnaman — APPROVED** [review]
👍

**Pull request merged**